paprika/data/cb6-but/cb6-but-dum.pdb

```diff
@@ -0,0 +1,131 @@
+ATOM      1  C   CB6     1      -0.519   4.975   1.835  1.00  0.00
```

Comment:
Why did this file change?

Reply:
The PDB file with the dummy atoms was not in the `data/cb6-but` folder. Not sure if it was there at one time before.

Reply:
Odd...